### PR TITLE
NEWS: Void Linux at 35c3

### PR DESCRIPTION
**Pull request by Gottox**
@void-linux/pkg-committers Please have a look and comment. I'm not a native speaker so bare with me :)

### CODE REVIEW

File: _posts/2018-12-22-35c3.markdown [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Void Linux at 35c3"
+comments: true
+---
+
+![35C3 Void](/assets/img/35c3-void.png)
+
+2018 has been an intense year for Void. There were many ups and downs. To wrap
+up the months behind we're meeting at the [35c3](https://events.ccc.de/category/congress/35c3/).
```
Comment (the-maldridge) [inline _posts/2018-12-22-35c3.markdown]: maybe swap this for "To wrap up the year we're meeting at..." as that reads a bit more naturally.

File: _posts/2018-12-22-35c3.markdown [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+For meeting other Voiders and chat about related and unrelated topics there's
+the
+[**Void Linux Assembly**](https://signup.c3assemblies.de/assembly/cf15ba5d-5bb1-473d-827c-b83862de75a1)
+
+Furthermore, on day 1 (2018-12-27) 19:00, [@Gottox](https://twitter.com/Gottox) is doing his
```
Comment (the-maldridge) [inline _posts/2018-12-22-35c3.markdown]: Furthermore would usually be a continuation of a singular idea, maybe use 'Also' instead.

File: _posts/2018-12-22-35c3.markdown [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+annual talk about [**The State Of the
+Void**](https://pretalx.35c3oio.freifunk.space/35c3oio/talk/XCU9PD/).
+
+After the talk, around 20:00, Void is inviting to a **Void User Meetup**. We'll
```
Comment (the-maldridge) [inline _posts/2018-12-22-35c3.markdown]: is inviting *you*

File: _posts/2018-12-22-35c3.markdown [hunk 4]
```diff
@@ -0,0 +1,26 @@
+---
+layout: post
+title: "Void Linux at 35c3"
+comments: true
+---
+
+![35C3 Void](/assets/img/35c3-void.png)
+
+2018 has been an intense year for Void. There were many ups and downs. To wrap
```
Comment (Vaelatern) [inline _posts/2018-12-22-35c3.markdown]: `2018 has been an intense year for Void, with many ups and downs.`

File: _posts/2018-12-22-35c3.markdown [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+up the months behind we're meeting at the [35c3](https://events.ccc.de/category/congress/35c3/).
+
+For meeting other Voiders and chat about related and unrelated topics there's
```
Comment (Vaelatern) [inline _posts/2018-12-22-35c3.markdown]: `To meet other Voiders and chat...`

File: _posts/2018-12-22-35c3.markdown [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+
+Furthermore, on day 1 (2018-12-27) 19:00, [@Gottox](https://twitter.com/Gottox) is doing his
+annual talk about [**The State Of the
+Void**](https://pretalx.35c3oio.freifunk.space/35c3oio/talk/XCU9PD/).
+
+After the talk, around 20:00, Void is inviting to a **Void User Meetup**. We'll
+meet at the Open Infrastructure Orbit in front of the lecture area, find the
+nearest bar, grab a Beer or a Tschunk, and get into interesting conversations.
+
+I'm looking forward to a great event with an awesome community!
```
Comment (Vaelatern) [inline _posts/2018-12-22-35c3.markdown]: `with our awesome community` might be better, but this can go either way.

**Pull request merged**